src/main/resources/edu/cornell/kfs/fp/cu-ojb-fp.xml

```diff
@@ -733,8 +733,53 @@
     </reference-descriptor>
 </class-descriptor>
 
-</descriptor-repository>
 
+  <class-descriptor class="edu.cornell.kfs.fp.businessobject.IncomingWireAchMapping" table="FP_INCOMING_WIRE_ACH_MAP_TC">
+    <field-descriptor name="shortDescription" column="SHORT_DESC" jdbc-type="VARCHAR" primarykey="true" index="true" />
```

Comment:
my understanding is yes it will be unique enough, and if they do have trouble, we'll work it out when the time comes